### PR TITLE
Fix Stylus configure dialog height by refactoring and Optimizing Stylus Popup Height Adjustment Logic

### PR DESCRIPTION
**Pull request by Ameer-Jamal**
### PR Description: Refactor and Optimize Stylus Popup Height Adjustment Logic

**Summary:**
This PR refactors the JavaScript logic responsible for adjusting the height of the `#stylus-popup` element within the configuration dialog. The changes align the code style with existing patterns in the project, enhance performance, and ensure maintainability.

**Key Changes:**
1. **Refactor to Match Existing Code Style:**
   - Replaced direct `addEventListener` usage with the more concise `document.on` method, in line with the existing codebase.
   - Utilized `$` as a shorthand for `document.querySelector` for consistent element selection.

2. **Performance Enhancements:**
   - Implemented a slight delay (`setTimeout` with 25ms) to ensure that necessary DOM elements are fully available before manipulation.
   - Optimized the calculation of the maximum height for the configuration dialog by iterating through all child elements and calculating the required height.

3. **Modularization:**
   - Split the logic into smaller, reusable functions: `handleConfigureClick`, `calculateDialogHeight`, `updateStylusPopupHeight`, and `resetStylusPopupHeight`. This improves readability and maintainability.

4. **Reset Logic:**
   - Added functionality to reset the height of the `#stylus-popup` when the `#message-box-close-icon` is clicked, ensuring the UI returns to its default state.

**Impact:**
- **Consistency**: The code is now stylistically consistent with the rest of the project, improving maintainability.
- **Performance**: Optimizations ensure that the code runs efficiently across millions of users, with minimal impact on page load and interaction times.
- **Maintainability**: The refactoring into functions allows for easier future updates and debugging.

**Testing:**
- Verified that the `#stylus-popup` height adjusts correctly when the configuration button is clicked.
- Ensured that the height resets appropriately when the close icon is clicked.

**Conclusion:**
This refactor ensures that the code not only performs well but also adheres to the project's coding standards, setting a solid foundation for future enhancements.

# Images : 
## Before  :
image

## After:

image

### CODE REVIEW

**Comment by Ameer-Jamal** [comment]
also the height isnt over done so you can scroll within the configure dialog

**Comment by Ameer-Jamal** [comment]
@tophf Can you check this, thank you!

**Comment by tophf** [comment]
The problem looks like #1795 which is already fixed in the **source code**, but it wasn't released yet. Could you test it?

**Comment by Ameer-Jamal** [comment]
@tophf  I will test it , what branch is the fix on ?

**Comment by tophf** [comment]
It's already in the default branch.

**Comment by Ameer-Jamal** [comment]
@tophf  Well in that case I forked master (which im assuming is the default) a couple hours ago, and the problem was still present

**Comment by tophf** [comment]
What's your OS, OS scaling, browser zoom in browser settings, extension zoom in Stylus style manager page?

**Comment by Ameer-Jamal** [comment]
> What's your OS, OS scaling, browser zoom in browser settings, extension zoom in Stylus style manager page?
@tophf 
Im on windows, I also have mac and have noticed the same problem, im on a 4k 32 inch screen my browser is edge and im usually on 100% zoom, I've tested multiple browsers zoom levels it makes no difference 

Windows zoom level and mac I have them at 125%

**Comment by tophf** [comment]
My display is single density and I can't reproduce the problem at 125%. Anyway, there should be no need to do what you did in this PR and recalculate positions. The problem should be fixed in our existing functions: `adjustSizeForPopup` (config-dialog.js) and `onWindowResize` (popup.js).

**Comment by tophf** [comment]
I see there was a related bug in our code. Try changing `all:initial` to `all:revert` in popup.js line 85.

**Comment by tophf** [comment]
Also please check `max-height` in the popup's `<body>` element in devtools i.e. right-click inside the popup and select "inspect" in the menu.

**Comment by Ameer-Jamal** [comment]
Oh i see yes I just tested and making all:intial go to all:revert fixed the issue ! 
My max height in body is    
 min-width: 325px !important;
    min-height: 543.636px !important;

**Comment by Ameer-Jamal** [comment]
@tophf I removed my change and updated the change

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "less": "^4.2.0",
     "lz-string-unsafe": "^1.4.4-fork-1",
     "stylelint-bundle": "^16.5.0",
+    "Stylus": "file:",
```
Comment (tophf) [inline package.json:19]: What's this? Shouldn't be necessary.

**Comment by Ameer-Jamal** [comment]
@toph my mistake those pesky json files commited by mistake i fixed it

**Pull request merged**